### DISCUSSION TITLE
Is it possible to combine multiple RTSP streams into a new stream with them aligned horizontally?

### DISCUSSION THREAD

**Discussion by julianrinaldi** [bluenviron/mediamtx#747] [Questions]
I just bought a Reolink Duo, and would like to combine the images from the left and right camera into a single stream. Is this possible with RTSP Simple Server?

**Comment by aler9** [comment]
Hello, you can do this with a video streaming software like OBS Studio, and then publish the result to rtsp-simple-server.

[a comment by an automated account is not shown]
